views/js/qtiCreator/editor/ckEditor/htmlEditor.js

```diff
@@ -233,7 +233,12 @@ define([
                     if(typeof options.qtiMedia !== 'undefined'){
                         ckConfig.qtiMedia = options.qtiMedia;
                     }
-
+                    if(typeof options.qtiImage !== 'undefined'){
```

Comment:
Are we strictly looking for `undefined`? At quick glance, this looks like all this can be replaced with a deep merge.